settings/src/script.js

```diff
@@ -128,7 +128,6 @@ function Main( { userId } ) {
 		screenContent = (
 			<Card>
 				<CardHeader className="wporg-2fa__navigation" size="xSmall">
-					<Flex>
```

Comment:
It looks like this was the only usage of `Flex`, so we can remove the `import` too.

Reply:
Ah, missed that, thanks.